docs/UserGuide.md

```diff
@@ -155,9 +204,10 @@ _Details coming soon ..._
 
 ## Command summary
 
-| Action   | Format                 | Examples                                                                                                    |
-|----------|------------------------|-------------------------------------------------------------------------------------------------------------|
-| Add      | `add c`, `add p`       | e.g., `add p n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney`;<br/>`add c n/Apple p/98765432 e/johnd@example.com d/Top tech companyt/tech t/interested`  |
-| Delete   | `delete c INDEX`, `delete p INDEX` | e.g., `delete p 3`                                                                                          |
-| List     | `list c`, `list p`     |                                                                                                             |
-| View     | `view c INDEX`, `view p INDEX`   | e.g., `view c 1`                                                                                            |
+| Action   | Format                 | Examples                                                                                                                                                                                            |
+|----------|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Add      | `add c`, `add p`       | e.g., `add p n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney`;<br/>`add c n/Apple p/98765432 e/johnd@example.com d/Top tech companyt/tech t/interested` |
```

Comment:
Just a nit here. Could you add a space between `d/Top tech company` and `t/tech` under the `add c` command example?